stock_product_pack/models/product_product.py

```diff
@@ -13,10 +13,10 @@ class ProductProduct(models.Model):
     def _compute_quantities_dict(
         self, lot_id, owner_id, package_id, from_date=False, to_date=False
     ):
-        res = super()._compute_quantities_dict(
+        packs = self.filtered("pack_ok")
```

Comment:
I think we could improve this, because self.filtered will browse all items, then the self - packs will browse again all items to make the computation. I think you can do the split inside this method to avoid to browse 2 times the elements.
Why do you think about it?

Reply:
The prefetch is shared, so not very big problem. There's just a new set operation.